### DISCUSSION TITLE
Add the requesting user as a tag when request is sent to sonarr/radarr

### DISCUSSION THREAD

**Discussion by mattkbach** [sct/overseerr#2443] [Feature Requests]
I think it would be great if we could have the option to set the tag as the username of the person who submitted the request. Or maybe more like a variable so Tag: overseer-{username}. When you are going through your lists of movies/tv shows in sonarr/radarr, you would be able to quickly see and sort by who requested what.

**Comment by echophex** [comment]
seconded.

**Comment by JonLaliberte** [comment]
Combined with https://github.com/sct/overseerr/discussions/2940 would be great to have.

**Comment by MonkeyCrumpets** [comment]
This would be fantastic when combined with Sonarr/Radarr's calendar tag filter. You could generate a personalised calendar for each user with only their requests on it.

**Comment by pistachiomatt** [comment]
I would also use tags to organise my Plex so that I don't see other people's downloads on the dashboard
